pkg/kubelet/rkt/rkt.go

```diff
@@ -2367,26 +2429,31 @@ func (r *Runtime) GetPodStatus(uid kubetypes.UID, name, namespace string) (*kube
 		}
 	}
 
-	// If we are running no-op network plugin, then get the pod IP from the rkt pod status.
-	if r.network.PluginName() == network.DefaultPluginName {
-		if latestPod != nil {
+	if latestPod != nil {
```

Comment:
Can we change to:
```go
if latestPod == nil {
    glog.Warningf()
    return podStatus, nil
}
```

So that we don't need the big `if` block.
}